src/main/java/com/paneedah/mwc/weapons/Guns.java

```diff
@@ -167,145 +167,173 @@ public class Guns {
+
+        M249 = new M249Factory().createGun(proxy);
         StonerA1 = new StonerA1Factory().createGun(proxy);
-        SSG08 = new SSG08Factory().createGun(proxy);
-        DSR1 = new DSR1Factory().createGun(proxy);
-        L96A1 = new L96A1Factory().createGun(proxy);
-        M200Intervention = new M200InterventionFactory().createGun(proxy);
-        AS50 = new AS50Factory().createGun(proxy);
-        M82 = new M82Factory().createGun(proxy);
-//        NTW20 = new NTW20Factory().createGun(proxy);
+        M134 = new M134Factory().createGun(proxy);
+
+        //REVOLVER
+        Python = new PythonFactory().createGun(proxy);
+        TaurusRagingHunter = new TaurusRagingHunterFactory().createGun(proxy);
+        SW500Magnum = new SW500MagnumFactory().createGun(proxy);
+        ChiappaRhino = new ChiappaRhinoFactory().createGun(proxy);
+
+        //SIDEARM2
+        APS = new APSFactory().createGun(proxy);
+        MakarovPM = new MakarovPMFactory().createGun(proxy);
+
+        DesertEagle = new DesertEagleFactory().createGun(proxy);
+        Glock18C = new Glock18CFactory().createGun(proxy);
+        Glock19 = new Glock19Factory().createGun(proxy);
+        FiveSeven = new FiveSevenFactory().createGun(proxy);
+        M9A1 = new M9A1Factory().createGun(proxy);
+        P226 = new P226Factory().createGun(proxy);
+        MP443 = new MP443Factory().createGun(proxy);
+        VP70 = new VP70Factory().createGun(proxy);
+        M17 = new M17Factory().createGun(proxy);
+
+        SCCYCPX2 = new SCCYCPX2Factory().createGun(proxy);
+        HK_P12 = new HK_P12Factory().createGun(proxy);
+
+        //SIDEARM1
+        MAS21 = new MAS21Factory().createGun(proxy);
+        G2Contender = new G2ContenderFactory().createGun(proxy);
+        M712 = new M712Factory().createGun(proxy);
+        M1911 = new M1911Factory().createGun(proxy);
+        BrowningHiPower = new BrowningHiPowerFactory().createGun(proxy);
+
+        //LAUNCHER
+        M202 = new M202Factory().createGun(proxy);
+        RPG7 = new RPG7Factory().createGun(proxy);
+
+        M32MGL = new M32MGLFactory().createGun(proxy);
         GL06 = new GL06Factory().createGun(proxy);
         M79 = new M79Factory().createGun(proxy);
-        M32MGL = new M32MGLFactory().createGun(proxy);
-        RPG7 = new RPG7Factory().createGun(proxy);
-        M202 = new M202Factory().createGun(proxy);
-        M134 = new M134Factory().createGun(proxy);
-//        NinthSin = new NinthSinFactory().createGun(proxy);
-        
         Chainsaw = new ChainsawFactory().createGun(proxy);
+        turretgun = new turretgunFactory().createGun(proxy);
+        turretgunupgraded = new turretgunupgradedFactory().createGun(proxy);
+        turretgunsilenced = new turretgunsilencedFactory().createGun(proxy);
+
+//        PGMHecateII = new PGMHecateIIFactory().createGun(proxy);
+//        NTW20 = new NTW20Factory().createGun(proxy);
+//        NinthSin = new NinthSinFactory().createGun(proxy);
```

Comment:
What is all this change for?
Have you test every aspect of it since there is a comment that says:

> Try not to change the order of the guns to ensure stable recipes